talosintelligence_connector.py

```diff
@@ -366,21 +484,55 @@ def initialize(self):
         config = self.get_config()
 
         def insert_newlines(string, every=64):
-            return "\n".join(
-                string[i: i + every] for i in range(0, len(string), every)
-            )
+            return '\n'.join(string[i:i + every] for i in range(0, len(string), every))
```

Comment:
Our flake8 checks disagree with other formatting tools on whether or not there should be a space after this colon. For this reason and for readability, can we replace this one-liner with a block?